cmd/boulder/main.go

```diff
@@ -85,9 +85,11 @@ var boulderUsage = fmt.Sprintf(`Usage: %s <subcommand> [flags]
 	core.Command())
 
 func main() {
-	var subcommand func()
+	var command string
 	if core.Command() == "boulder" {
-		// The operator is running the boulder binary directly.
+		// Operator passed the boulder component as a subcommand.
+		command = os.Args[1]
```

Comment:
I'd do this assignment on line 113, after the len/help/list checks. This var only gets used if those returns all get skipped, and it's kinda confusing to assing os.Args[1] to a variable, then reference os.Args[1] a bunch of times below rather than referencing the variable.